### DISCUSSION TITLE
Add Stochastic gradient HMC

### DISCUSSION THREAD

**Discussion by rlouf** [blackjax-devs/blackjax#93] [New features]
https://arxiv.org/abs/1402.4102

**Comment by junpenglao** [comment]
🤖 *This reply was drafted with Claude and reviewed by me (@junpenglao).*

Implemented as `blackjax.sghmc` (alongside `sgld`, `sgnht`, `csgld`). Closing.
